src/staticanalysis/bot/static_analysis_bot/clang/tidy.py

```diff
@@ -308,13 +315,16 @@ def as_text(self):
         # Always add body as it's been cleaned up
         if self.body:
             body += '\n{}'.format(self.body)
+        if self.reason:
+            body += '\n{}'.format(self.reason)
 
         return body
 
     def as_markdown(self):
         return ISSUE_MARKDOWN.format(
             type=self.type,
             message=self.message,
+            reason=self.reason,
```

Comment:
Nit: Please move `reason` just below the `body`. I feel it would make more sense there.